src/lib/containers/access_requirements/HasAccess.stories.tsx

```diff
@@ -4,7 +4,7 @@ import { HasAccessV2 } from './HasAccessV2'
 
 // More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
 export default {
-  title: 'Components/HasAccess',
+  title: 'Governance/HasAccess',
```

Comment:
Moving all Governance component stories to top level "Governance" Storybook folder